src/PostgresEventStore.php

```diff
@@ -308,42 +309,43 @@ public function loadReverse(
 
         $tableName = $this->persistenceStrategy->generateTableName($streamName);
 
-        $sql = [
-            'from' => "SELECT * FROM $tableName",
-            'orderBy' => 'ORDER BY no DESC',
-        ];
+        $where = [];
+        $values = [];
 
-        foreach ($metadataMatcher->data() as $match) {
+        foreach ($metadataMatcher->data() as $key => $match) {
             $field = $match['field'];
             $operator = $match['operator']->getValue();
             $value = $match['value'];
+            $parameter = ':metadata_'.$key;
 
             if (is_bool($value)) {
-                $value = var_export($value, true);
-                $sql['where'][] = "metadata ->> '$field' $operator '$value'";
-            } elseif (is_string($value)) {
-                $value = $this->connection->quote($value);
-                $sql['where'][] = "metadata ->> '$field' $operator $value";
-            } else {
-                $sql['where'][] = "metadata ->> '$field' $operator '$value'";
+                $where[] = "metadata->'$field' $operator '".var_export($value, true)."'";
```

Comment:
`metadata ->>` instead of `metadata -> `

see: https://www.postgresql.org/docs/9.3/static/functions-json.html

`->` means: Get JSON object field
`->>` means: Get JSON object field as text